src/scripts/types/gallery-item.ts

```diff
@@ -26,6 +26,7 @@ export interface VideoItem {
 
 export interface EmbeddedItem {
   type: GalleryItemType.Embedded;
+  text?: string;
```

Comment:
Maybe better `description` 🤔

Reply:
It was `text` before, that's we I kept it. But might be a good opportunity to use something more descriptive :)

Reply:
You are right, for other gallery types it's also text. So let's just keep that.